blackhole.c

```diff
@@ -268,6 +270,66 @@ static void blackhole_restore_reset_state(struct tenstorrent_device *tt_dev) {
 	noc_write32(bh, x, y, PCIE_DBI_ADDR + DBI_DEVICE_CONTROL_DEVICE_STATUS, device_control);
 }
 
+static ssize_t bh_show_pcie_single_counter(struct device *dev, char *buf, u32 counter_offset, int noc)
+{
+	struct tenstorrent_device *tt_dev = dev_get_drvdata(dev);
+	struct blackhole_device *bh = tt_dev_to_bh_dev(tt_dev);
+	u64 offset = NOC_STATUS_OFFSET + (4 * counter_offset) + (noc * NOC1_NOC2AXI_OFFSET);
+	u32 value = ioread32(bh->noc2axi_cfg + offset);
+	return scnprintf(buf, PAGE_SIZE, "%u\n", value);
+}
+
+#define BH_PCIE_COUNTER_ATTR_RO_PAIR(_base_name_str, _counter_type_id_const) \
```

Comment:
A comment to say the pair are counters for noc0 and noc1, to save future reviewers having to work it out.